DependencyInjection/Configuration.php

```diff
@@ -245,12 +246,17 @@ private function configureDbalDriverNode(ArrayNodeDefinition $node): void
         $node
             ->validate()
             ->always(static function (array $values) {
-                $deprecatedOptions = ['override_url' => true];
-                $deprecatedValues  = array_intersect_key($values, $deprecatedOptions);
+                if (! isset($values['url'])) {
+                    return $values;
+                }
+
+                $urlConflictingOptions = ['host' => true, 'port' => true, 'user' => true, 'password' => true, 'path' => true, 'dbname' => true, 'unix_socket' => true, 'memory' => true];
```

Comment:
I think driver is missing, isn't it?

Reply:
`driver` is in another branch of the config tree.
I don't think that's an issue.